src/chunk.c

```diff
@@ -1890,7 +1891,52 @@ ts_chunk_drop_chunks(PG_FUNCTION_ARGS)
 	foreach(lc, ht_oids)
 	{
 		Oid			table_relid = lfirst_oid(lc);
+		List	   *fk_relids = NIL;
+		ListCell   *lf;
+
+		/* get foreign key tables associated with the hypertable */
+		{
```

Comment:
I might consider making this it's own static function that returns a list given that it's a block anyway. But up to you.